datasets/afvalwijzer/dataset.json

```
@@ -0,0 +1,227 @@
+{
+    "id": "afvalwijzer",
+    "type": "dataset",
+    "authorizationGrantor": "n.v.t.",
+    "theme": [
+        "Duurzaamheid en milieu, wonen"
+    ],
+    "homepage": "https://data.amsterdam.nl",
+    "owner": "Gemeente Amsterdam, Afval en Grondstoffen",
+    "dateModified": "",
+    "spatialDescription": "Amsterdam",
+    "version": "1.0.0",
+    "title": "Afvalwijzer",
+    "language": "nl",
+    "dateCreated": "2022-01-01T00:00:00",

```

This works locally, let me try to commit it:

```suggestion
    "dateCreated": "2022-01-01T00:00:00+01:00",

```